src/main/java/MainWindow.java

```diff
@@ -0,0 +1,51 @@
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
```

Comment:
Maybe it would be better to leave a line between your imports and javadocs? Might be better to standardize throughout your code.